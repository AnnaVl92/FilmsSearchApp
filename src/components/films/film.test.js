import React from 'react';
import { mount } from 'enzyme';
import Film from './film.jsx';
import expect from 'expect';

// eslint-disable-next-line no-undef
itx('renders correctly', () => {
	const mockFilm = {
		img: 'https://vignette.wikia.nocookie.net/cinemorgue/images/9/98/Kill_bill2_01.jpg/revision/latest?cb=20151221212736',
		title: 'Kill Bill: Volume 2',
		releaseDate: 2004,
		genre: 'Action & Adventure'
	};
	const component = mount(<Film film={mockFilm} />);
	expect(component).toMatchSnapshot();
});
