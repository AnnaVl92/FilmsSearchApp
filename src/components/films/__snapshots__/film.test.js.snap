// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Film
  film={
    Object {
      "genre": "Action & Adventure",
      "img": "https://vignette.wikia.nocookie.net/cinemorgue/images/9/98/Kill_bill2_01.jpg/revision/latest?cb=20151221212736",
      "releaseDate": 2004,
      "title": "Kill Bill: Volume 2",
    }
  }
>
  <div
    className="col-md-4"
  >
    <div
      className="card film"
    >
      <div
        className="film-img-cnt"
      >
        <img
          alt=""
          className="card-img-top"
          src="https://vignette.wikia.nocookie.net/cinemorgue/images/9/98/Kill_bill2_01.jpg/revision/latest?cb=20151221212736"
        />
      </div>
      <div
        className="card-body"
      >
        <h2
          className="card-title"
        >
          Kill Bill: Volume 2
        </h2>
        <p
          className="film-year"
        >
          2004
        </p>
        <p
          className="film-genre"
        >
          Action & Adventure
        </p>
      </div>
    </div>
  </div>
</Film>
`;
